2/README.md

```diff
@@ -0,0 +1,122 @@
+
+This specification is based on the original [Digistan 1/COSS](http://www.digistan.org/spec:1/COSS)
+
+
+## Change Process
+This document is governed by the [2/COSS](../2/README.md) (COSS).
+
+## Language
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119)
+
+## Goals
+The primary goal of COSS is to facilitate the process of writing, proving, and improving new technical specifications. A "technical specification" defines a protocol, a process, an API, a use of language, a methodology, or any other aspect of a technical environment that can usefully be documented for the purposes of technical or social interoperability.
+
+COSS is intended to above all be economical and rapid, so that it is useful to small teams with little time to spend on more formal processes.
+
+Principles:
+
+* We aim for rough consensus and running code.
+* Specifications are small pieces, made by small teams.
+* Specifications should have a clearly responsible editor.
+* The process should be visible, objective, and accessible to anyone.
+* The process should clearly separate experiments from solutions.
+* The process should allow deprecation of old specifications.
+
+Specifications should take minutes to explain, hours to design, days to write, weeks to prove, months to become mature, and years to replace.
+
+Specifications have no special status except that accorded by the community.
+
+## Architecture
+COSS is designed around fast, easy to use communications tools. Primarily, COSS uses a wiki model for editing and publishing specifications texts.
+
+* The *domain* is the conservancy for a set of specifications in a certain area.
+* Each domain is implemented as an Internet domain, hosting a wiki and optionally other communications tools.
+* Each specification is a set of wiki pages, together with comments, attached files, and other resources.
+* Important specifications may also exist as subdomains, i.e. child wikis.
+
+Individuals can become members of the domain by completing the necessary legal clearance. The copyright, patent, and trademark policies of the domain must be clarified in an Intellectual Property policy that applies to the domain.
+
+Specifications exist as multiple pages, one page per version of the specification (see "Branching and Merging", below), which may be assigned URIs that include an incremental number. Thus, we refer to a specification by specifying its domain, number, and short name. New versions of the same specification will have new numbers. The syntax for a specification reference is:
+
+
+  <domain>/spec:<number>/<shortname>
+
+
+For example, this specification is **rfc.unprotocols.org/spec:2/COSS**. The short form **2/COSS** may be used when referring to the specification from other specifications in the same domain.
```

Comment:
I'd simplify all of this saying that every specification lives in a directory under the GitHub repository.

Reply:
Maybe the directory naming conventions should also be defined? we could adapt the above convention in a directory naming scheme?

Reply:
I agree, my idea is to have something simple, like
- `bigchaindb/rfc` this repository
  - `README.md` with pointers to C4 and COSS
  - `1/` directory for RFC 1
    - `README.md` the actual specification
    - `...` all other media files needed.
  - `2/` directory for RFC 2
    - `README.md` the actual specification
    - `...` all other media files needed.

Reply:
Is the idea to name each directory 1, 2, 3, etc. --- one per specification?

Reply:
In my opinion yes, one per specification. My suggestion is to name the main specification `README.md`, so GitHub shows it right away.

Reply:
I just noticed that an EIP takes the number of the PR. We can do the same.